### PR TITLE
Recommand recent genomes

### PR DESCRIPTION
**Pull request by MaxUlysse**
_ie_ `GRCh38` instead of `GRCh37`

### CODE REVIEW

File: markdown/developers/guidelines.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ If possible, it's great if pipelines can also have:
   * Nearly all nf-core pipelines use a conda `env` script to list their software requirements.
     The pipeline Docker images are then built using this, meaning that with a single file your pipeline can support nextflow users running with conda, docker or singularity.
   * The [nf-core template](/tools#creating-a-new-workflow) comes with all required code to support this setup.
+* All pipelines should use by default a recent reference genomes (_ie_ `GRCh38` for human...)
```
Comment (apeltzer) [inline markdown/developers/guidelines.md]:
```suggestion
* All pipelines should attempt to use a recent reference genome draft(_ie_ `GRCh38` for human...)
```

**Comment by ewels** [comment]
I'm a bit confused by this now that I see it. Isn't the choice of reference genome up to the user? The guidelines are more for pipeline developers..

**Comment by MaxUlysse** [comment]
I think it should be there.
It's something that we developers should push, using more recent genomes, especially GRCh38 over GRCh37

**Comment by ewels** [comment]
But it's not up to us developers what people use? I guess this comes down to a minor wording issue maybe. That pipelines should _support the use_ of recent reference genomes..

**Comment by apeltzer** [comment]
Not sure its just for the enduser: Some pipelines we have are relatively tight to use e.g. only GRCH37, GRCH38 supported by downstream tools (`epitopeprediction` for example). 

And I think the statement here is relatively open:

```
 * All pipelines should attempt to use a recent reference genome draft(_ie_ `GRCh38` for human...) 
```

"Should attempt ..." is not enforcing it but just stating that you should aim for current drafts of references when implementing your pipeline. Could be maybe better formulated I agree ...

**Comment by ewels** [comment]
Yeah, I think we're all agreeing here. I was just confused by the wording and thought it meant something else at first.

**Pull request merged**